src/init.cpp

```diff
@@ -199,7 +202,9 @@ pika::program_options::options_description getOptionsDescription() {
   // Tune parameters command line options
   desc.add_options()(
       "dlaf:red2band-panel-nworkers", pika::program_options::value<std::size_t>(),
-      "Maximum number of threads to use for computing the panel in the reduction to band algorithm.");
+      "The maximum number of threads to use for computing the panel in the reduction to band algorithm.");
```

Comment:
The added `The` at the beginning is only for consistency with the description in `dlaf/tune.h` and the other options, but it doesn't really belong in this PR. Happy to move it out if you prefer that.